include/open62541/client_subscriptions.h

```diff
@@ -78,6 +82,24 @@ UA_Client_Subscriptions_create_async(UA_Client *client,
     UA_ClientAsyncServiceCallback callback,
     void *userdata, UA_UInt32 *requestId);
 
+UA_CreateSubscriptionResponse UA_EXPORT UA_THREADSAFE
+UA_Client_Subscriptions_create_complete_data_change(UA_Client *client,
```

Comment:
We don't use snake_case_notation in the library.
Use camlCaseNotation instead.

Reply:
I renamed the functions to `UA_Client_Subscriptions_createCompleteDataChange` and `UA_Client_Subscriptions_createCompleteDataChange_async` is this OK?